tests/functions_tests/test_loss.py

```diff
@@ -0,0 +1,38 @@
+# Currently, the test for SoftmaxCrossEntropy is disabled since onnxruntime
+# does not support OneHot node. After OneHot node becomes available, we may
+# use the comment-out part for testing.
+
+# import unittest
+#
+# import chainer
+# from chainer import testing
+# import numpy as np
+#
+# from onnx_chainer.testing import test_onnxruntime
+#
+#
+# @testing.parameterize(
+#     {'in_shape': (3, 5)},
+# )
+# class TestSoftmaxCrossEntropy(unittest.TestCase):
```

Comment:
I prefer to use `@unittest.skip("demonstrating skipping")` decorator than comment out.